src/Command.js

```diff
@@ -166,6 +207,19 @@ export default class Command {
     }
   }
 
+  _legacyOptions() {
+    let opts = {};
+    if (this.name === "bootstrap" && (this.repository.asiniJson || {}).bootstrapConfig) {
```

Comment:
Yeah, I wavered on that.  Wasn't sure if there would ever be a case where we cared whether it was even defined.  I guess probably not.  I'll clean this up.